### PR TITLE
Ensure mobile window content enqueues a refresh event on show

### PR DESCRIPTION
**Pull request by fpabl0**
### Description:

Fixes issue running fyne_demo with mobile tag, reported on Slack.

### Checklist:

- [ ] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

File: internal/widget/scroller.go [hunk 1]
```diff
@@ -402,6 +402,9 @@ func (r *scrollContainerRenderer) handleShadowVisibility(offset, contentSize, sc
 }
 
 func (r *scrollContainerRenderer) updatePosition() {
+	if r.scroll.Content == nil {
```
Comment (fpabl0) [inline internal/widget/scroller.go:405]: While trying to fix the bug about `window.Show`, I found this other bug, that causes a crash in fyne_demo with mobile tag, not sure why it worked before. But it seems to be unrelated to the PR #1986

**Review by andydotxyz — APPROVED** [review]

**Comment by andydotxyz** [comment]
Thanks, on develop and release branches
